src/manifest/dbtProject.ts

```diff
@@ -1011,6 +1040,14 @@ select * from renamed
     );
   }
 
+  async findPackagesInstallPath() {
+    const packages_install_path = await this.python?.lock(
+      (python) => python!`to_dict(project.config.packages_install_path)`,
+    );
+    console.log(`Packages install path: ${packages_install_path}`);
```

Comment:
Remove this line as this log line is not super useful, we would not know the packages install path belongs to which project for example.